nwnltr.c

```diff
@@ -46,6 +46,7 @@ struct cfg {
 #define die(format, ...)                                \
     do {                                                \
         fprintf(stderr, format "\n", ##__VA_ARGS__);    \
+        fflush(stderr);                                 \
```

Comment:
This is super weird, since stderr is supposed to be line-buffered, which means it's always flushed on `\n`. Doesn't hurt though.

Reply:
The way the later versions (since about 5 years ago) of mingw32/msys2 on windows implement stdout/stderr/stdin is a little screwy (ends up with gdb's exit prompt always auto-pressing Y and a few other weird things going on). I don't think its strictly necessary at all, but it doesn't seem to hurt. I can get rid of it, if needed.